### ISSUE TITLE
[Feature Request] Autoupdate timeline

### ISSUE THREAD

**Issue by ferlanero** [Wunderfitz/harbour-piepmatz#40]
As the excellent Twitter client Piepmatz is, it deserves best features. I think that autoupdate timeline while you are inside application, is a must to an it adds easy to use to Piepmatz

Anyway, thanks for all your efforts Sebastian!

**Comment by Wunderfitz** [comment]
Thanks again! 

Well, the basic feature is quite easy to implement, you are right! However, I'm still struggling a bit to update the content of the timeline so that the cursor position stays _exactly_ - so pixel-exactly - on the position as it was before the update. Only if this is achievable I would integrate such an auto-update without explicit user interaction. Otherwise it could be quite a distracting user experience if the item which I'm currently reading is jumping back and forth - even if it's just a bit (as it happens currently when you explicitly press "Update"). 

Most certainly, this won't be possible before I included content caching as I heavily use lazy-loading for link previews, attached tweets etc. This lazy loading simply influences the current cursor position as well. If everything is cached, I can handle that much better. 

To sum it up: That's definitely on my list. However, it might not come before I implemented some other features. Next on the list are:
- Download Media (you already tested that ;))
- Delete tweet (already implemented)
- Notifications for people retweeting your content and people following you
- List support

Best regards and have fun with Piepmatz! Sebastian
